package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,777 @@
+              <donor index (0-based)>,
+              <acceptor index (0-based)>, <donor string>, <acceptor string>,
+              <distance>, <angle>
+           ],
+
+           # hbonds linking the selection 1 and selection 2 to the bridgeing water 2
+           [ # hbond 1 from selection 1 to the bridging water 2
+              <donor index (0-based)>,
+              <acceptor index (0-based)>, <donor string>, <acceptor string>,
+              <distance>, <angle>
+           ],
+           [ # hbond 1 from selection 2 to the bridging water 2
+              <donor index (0-based)>,
+              <acceptor index (0-based)>, <donor string>, <acceptor string>,
+              <distance>, <angle>
+           ],
+           ....
+        ],
+        [ # frame 2
+          [ ... ], [ ... ], ...
+        ],
+        ...
+    ]
+
+Using the :meth:`WaterBridgeAnalysis.generate_table` method one can reformat
+the results as a flat "normalised" table that is easier to import into a
+database or dataframe for further processing.
+:meth:`WaterBridgeAnalysis.save_table` saves the table to a pickled file. The
+table itself is a :class:`numpy.recarray`.
+
+Detection of water bridges
+---------------------------
+Water bridges are recorded if a bridging water simultaneously form two hydrogen bonds with selection 1 and selection 2.
+
+Hydrogen bonds are detected as is described in \ :class:`~MDAnalysis.analysis.hbonds.hbond_analysis.HydrogenBondAnalysis`, see \ :ref:`Detection-of-hydrogen-bonds`.
+
+The lists of donor and acceptor names can be extended by providing lists of
+atom names in the `donors` and `acceptors` keywords to
+:class:`WaterBridgeAnalysis`. If the lists are entirely inappropriate
+(e.g. when analysing simulations done with a force field that uses very
+different atom names) then one should either use the value "other" for `forcefield`
+to set no default values, or derive a new class and set the default list oneself::
+
+ class WaterBridgeAnalysis_OtherFF(WaterBridgeAnalysis):
+       DEFAULT_DONORS = {"OtherFF": tuple(set([...]))}
+       DEFAULT_ACCEPTORS = {"OtherFF": tuple(set([...]))}
+
+Then simply use the new class instead of the parent class and call it with
+`forcefield` = "OtherFF". Please also consider to contribute the list of heavy
+atom names to MDAnalysis.
+
+How to perform WaterBridgeAnalysis
+-----------------------------------
+
+All water bridges between arginine and aspartic acid can be analysed with ::
+
+  import MDAnalysis
+  import MDAnalysis.analysis.hbonds
+
+  u = MDAnalysis.Universe('topology', 'trajectory')
+  w = MDAnalysis.analysis.hbonds.WaterBridgeAnalysis(u, 'resname ARG', 'resname ASP')
+  w.run()
+
+The results are stored as the attribute
+:attr:`WaterBridgeAnalysis.timeseries`; see :ref:`Analysis Output` for the
+format.
+
+An example of using the :attr:`~WaterBridgeAnalysis.timeseries` would be detecting the precentage of time a certain water bridge exits. ::
```

Comment:
percentage

Reply:
Break lines at 79 characters. See the [Style Guide](https://github.com/MDAnalysis/mdanalysis/wiki/Style-Guide#important-requirements).

Reply:
@orbeckst Sorry, I will make sure all the words are within the boundary in the next commit.